packages/core/tests/acceptance/table-test.ts

```diff
@@ -78,8 +78,8 @@ module('Acceptance | table', function(hooks) {
     await visit('/table');
 
     await click('.table-config__total-toggle-button .x-toggle-btn');
-    await fillIn('.timeDimension > .table-header-cell__input', 'test');
-    await blur('.timeDimension > .table-header-cell__input');
+    await fillIn('.timeDimension > .input > .table-header-cell__input', 'test');
```

Comment:
do we need this extra class?

Does ths work?
```suggestion
    await fillIn('.timeDimension .table-header-cell__input', 'test');
```

Reply:
I had to add this for the tests to work. I used the inspect element and table header cell input was nested within input